src/main/scala/verifier/DefaultMainVerifier.scala

```diff
@@ -513,4 +517,12 @@ class DefaultMainVerifier(config: Config,
     }
     results
   }
+
+  /**
+    * In case Silicon encounters an expected error (i.e. `ErrorMessage.isExpected`), Silicon continues (until at most
+    * config.numberOfErrorsToReport() have been encountered (per member)).
+    * This function retrieves all errors that have been encountered and creates a verification result containing them.
```

Comment:
The last sentence doesn't seem fitting:
1. `retrieves all errors` doesn't obviously match a single `::` concatenation
2. `creates a verification result`, but return type is `Seq[VerificationResult]`